app/inventory/route.js

```diff
@@ -1,14 +1,16 @@
+import { translationMacro as t } from 'ember-i18n';
 import AbstractModuleRoute from 'hospitalrun/routes/abstract-module-route';
 import FulfillRequest from 'hospitalrun/mixins/fulfill-request';
 import InventoryId from 'hospitalrun/mixins/inventory-id';
 import InventoryLocations from 'hospitalrun/mixins/inventory-locations'; // inventory-locations mixin is needed for fulfill-request mixin!
 export default AbstractModuleRoute.extend(FulfillRequest, InventoryId, InventoryLocations, {
   addCapability: 'add_inventory_item',
+  buttonText: t('navigation.subnav.inventoryReceived'),
```

Comment:
This buttonText should use a separate localization entry, not reuse the one from the subnav (the text is intentionally lowercase for the button).